src/app/RecordingMetadata/LabelCell.tsx

```diff
@@ -36,21 +36,56 @@
  * SOFTWARE.
  */
 
+import { getLabelDisplay } from '@app/Recordings/Filters/LabelFilter';
+import { UpdateFilterOptions } from '@app/Shared/Redux/RecordingFilterReducer';
 import { Label, Text } from '@patternfly/react-core';
 import React from 'react';
+import { ClickableLabel } from './ClickableLabel';
 import { RecordingLabel } from './RecordingLabel';
 
 export interface LabelCellProps {
+  target: string,
   labels: RecordingLabel[];
+  // Must be specified along with updateFilters.
```

Comment:
Are these two props expected to always both be undefined or both be defined? If so, would it make sense to wrap them both into an optional config object, where the config object is optional but both of these fields within are required? This way the type system helps ensure the programmer doesn't make a mistake and only define one of the two.

Reply:
Right make sense!